js/PanoViewer.js

```diff
@@ -0,0 +1,379 @@
+
+function PanoViewer (opts) {
+    const self = this;
+    // Default Configuration
+    const config = {
+        API_ROOT: 'https://acc.api.data.amsterdam.nl/',
+        PANO_URI: 'panorama/recente_opnames/alle/',
+        RADIUS: 100,
+        FOV: 80,
+        MAX_FOV: 90,
+        MAX_RESOLUTION: 12 * 1024,
+        CAMERA_HEIGHT: 1.8,
+        LEVEL_PROPERTIES_LIST: [
+            {tileSize: 256, size: 256, fallbackOnly: true},
+            {tileSize: 512, size: 512},
+            {tileSize: 512, size: 1024},
+            {tileSize: 512, size: 2048}
+        ],
+        ADD_CALLBACKS: false,  // Wether to add callbacks on changes
+        CALLBACKS: {
+            'active': console.log,
+            'inactive': console.log
+        }
+    };
+
+    const viewerOpts = {
+        controls: {
+            mouseViewMode: 'drag'    // drag|qtvr
+        },
+        stageType: null,
+        stage: {
+            preserveDrawingBuffer: true
+        }
+    };
+    const degree_to_radian = Math.PI / 180;
+    let viewer, view, scene;  // Will contain the viewer
+
+    const loadScene = function (heading, pitch, fov, data) {
+        const image = data.image;
+        const hotspots = data.hotspots;
+        try {
+            const source = Marzipano.ImageUrlSource.fromString(
+                image.pattern, {cubeMapPreviewUrl: image.preview}
+            );
+            const viewLimiter = Marzipano.RectilinearView.limit.traditional(
+                config.MAX_RESOLUTION,
+                degree_to_radian * config.MAX_FOV
+            );
+
+            view = new Marzipano.RectilinearView({}, viewLimiter);
```

Comment:
pass meteen de yaw, pitch and fov: www.marzipano.net/reference/RectilinearView.html